### PR TITLE
Update static-files.rst

### PR DESCRIPTION
**Pull request by aftab10662**
Microsoft.AspNet.StaticFiles nuget package name is no where mentioned in Article.

I was looking for help on how to serve static assets from ASP.net Core App. After reading help document i get to know all extension method but could not see package name anywhere, again i found my self helpless.

Developers will get access to this Middleware if they install this specific nuget package.
.

### CODE REVIEW

File: aspnet/fundamentals/static-files.rst [hunk 1]
```diff
@@ -23,7 +23,7 @@ Static files can be stored in any folder under the webroot and accessed with a r
 
   http://<yourApp>/images/<imageFileName>
 
-In order for static files to be served, you must configure the :doc:`middleware` to add static files to the pipeline. This is accomplished by calling the ``UseStaticFiles`` extension method from  ``Startup.Configure`` as follows:
+In order for static files to be served, you must configure the :doc:`middleware` to add static files to the pipeline. This specific middleware can be configured by downloading Microsoft.AspNet.StaticFiles nuget package and by calling the ``UseStaticFiles`` extension method from  ``Startup.Configure`` as follows:
```
Comment (danroth27) [inline aspnet/fundamentals/static-files.rst]: How about this?: "This specific middleware can be configured by adding a dependency on the Microsoft.AspNetCore.StaticFiles package to your project and then calling the `UseStaticFiles` extension method from `Startup.Configure` as follows:"

**Comment by danroth27** [comment]
Thanks for this contribution!

We'd like this change to go into dev instead of master as we will be shipping RC2 shortly with the updated package names. Could you please rebase this PR on dev?

File: aspnet/fundamentals/static-files.rst (hunk 1 above)
Comment (aftab10662) [inline aspnet/fundamentals/static-files.rst]: Yes, it is more accurate and intention revealing.
Will do rebase on dev branch. Thank you.

**Comment by danroth27** [comment]
I'll take care of this for you via https://github.com/aspnet/Docs/pull/1221

**Comment by aftab10662** [comment]
Thank you.

**Pull request closed without merging**